ecommerce/tasks.py

```diff
@@ -0,0 +1,46 @@
+"""
+Ecommerce tasks
+"""
+from ecommerce.hubspot_api import (
+    send_hubspot_request,
+    make_contact_sync_message,
+    make_product_sync_message,
+    make_deal_sync_message,
+    make_line_item_sync_message,
+)
+from mitxpro.celery import app
+
+
+HUBSPOT_SYNC_URL = "/extensions/ecomm/v1/sync-messages"
+
+
+@app.task
+def sync_contact_with_hubspot(user_id):
```

Comment:
Can you add unit tests for these (verify that a request is made to the correct url with the correct data).